content/articles/an-extensive-guide-on-working-with-images-in-django/index.md

```diff
@@ -0,0 +1,424 @@
+Working with images is an important skill to have as a web developer and since Django is one of the most popular web development frameworks, it provides ways of dealing with images conveniently.
+
+This tutorial will look at various situations that you might encounter while working with images in Django and also provide guidance on how to tackle those situations.
```

Comment:
It would be great if you could add 1 - 2 lines about the "situations" that you are going to discuss.